atd-vze/src/Components/gridTableHelpers.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import get from "lodash.get";
+import { Alert } from "reactstrap";
+
+export const renderAlert = alert => <Alert color="warning">{alert}</Alert>;
+
+export const getChartData = (chartData, config) => {
+  // Look for records that match labels and increment total
+  const data = config.labels.map(type => {
+    let typeTotal = 0;
+    chartData[config.table].forEach(record => {
+      // If record is a string, increment total if match
+      if (config.isSingleRecord) {
```

Comment:
you could get rid of one level of nested `if`s if line 13 were just: 
`if(config.isSingleRecord && record[config.nestedKey][config.nestedPath] === type) {`

not sure that makes the code that much better 🤷‍♂